network-primitives/bootstrap_node_provisioner.tf

```diff
@@ -14,14 +14,23 @@ resource "null_resource" "boostrap-node-keys" {
     cluster_instance_ipv4s = join(",", local.bootstrap_nodes_ip_v4)
   }
 
-  # generate rpc node keys
+  # generate node keys
   provisioner "local-exec" {
     command     = "${var.path-to-scripts}/generate_node_keys.sh ${length(local.bootstrap_nodes_ip_v4)} ./bootstrap_node_keys.txt"
     interpreter = ["/bin/bash", "-c"]
     environment = {
       NODE_PUBLIC_IPS = join(",", local.bootstrap_nodes_ip_v4)
     }
   }
+
+  # generate boostrap node keys
```

Comment:
I don't think we use this currently, right? Or at least we're not using port 50000.